store-implementation/proxy-store/src/test/java/uk/gov/gchq/gaffer/proxystore/integration/ProxyStoreResponseDeserialiserIT.java

```diff
@@ -53,11 +54,9 @@ public class ProxyStoreResponseDeserialiserIT {
 
     private static final RestApiTestClient CLIENT = new RestApiV2TestClient();
 
-    @TempDir
-    public final File testFolder = CommonTestConstants.TMP_DIRECTORY;
```

Comment:
Is this incorrect everywhere it is used? Because I think there are other uses

Reply:
If it is marked as `final`, then JUnit considers that an error which justifies failing the test. Having it set to a value is incorrect as the value is set by JUnit and is actually set to the same thing as `CommonTestConstants.TMP_DIRECTORY` anyway. Having the value as public is not required either. There are no other instances in the codebase where it is marked as `final`, there are some where it is assigned a value, but that's not the problem here and would be more of a general tidy up task to fix.

This is an example of how it should be done if the temp directory is going to be used/reused in multiple tests in that class: https://github.com/gchq/Gaffer/blob/71199d3ebfa403404ce23d4eb5e0d4151dd38e89/core/operation/src/test/java/uk/gov/gchq/gaffer/export/LocalFileExporterTest.java#L49-L54

In this PR it was used in only one test and so could be moved into the test method as a parameter instead. This way it will be cleaned up once that individual test is run.